guillotina/json/serialize_content.py

```diff
@@ -56,20 +56,24 @@ def __init__(self, context, request):
         else:
             parent_summary = {}
 
+        factory = get_cached_factory(self.context.type_name)
+        behaviors = []
+        for behavior_schema in factory.behaviors or ():
+            behaviors.append(behavior_schema)
+
         result = {
             '@id': IAbsoluteURL(self.context, self.request)(),
             '@type': self.context.type_name,
             '@name': self.context.__name__,
             '@uid': self.context.uuid,
+            '@sb': behaviors,
```

Comment:
sorry, can we just call it "@static_behaviors"?

Reply:
this information is going to be on all responses of GET so I was willing to keep it short

Reply:
eh, I don't know man. I think we need the name to be descriptive.

Up to you now. I'll approve but you're free to make final decision.